dev-python/nbdime/nbdime-3.1.1-r2.ebuild

```
@@ -73,6 +76,7 @@ EPYTEST_DESELECT=(
 
 src_prepare() {
 	edos2unix \
+		setupbase.py \
```

Oh, so you're doing this because of patches? Could you do all installed files for consistency then? I suppose a bunch of `*`s will be necessary ;-).

---

Keeping CRLF won't affect users, right? I don't feel the need to format all installed files...

Also, do you think it's ok to format all source files with a find script?



```suggestion
                $(find . -type f -exec file "{}" ";" | grep CRLF | awk -F':' '{print $1}')

        distutils-r1_src_prepare
}
```

---

I'd rather go with `find -type -f -exec sed -i -e ... {} +`. `edos2unix` is so trivial there's no harm inlining the method.